### ISSUE TITLE
Camera gets reenabled in away mode

### ISSUE THREAD

**Issue by phoegon** [thecodingmachine/workadventure#2936]
When I'm in away mode my camera get's turned on when a colleague steps next to me.

It would be really nice if there was a setting, so the camera would only activate when I return to the tab, or maybe even only when I enable it. 

I've walked next to colleagues who had their volume turned down and it feels a bit creepy to watch them when they can't notice that I can see them.

As a workaround I usually turn my camera and microphone off and reenable them when a colleague walks up, but it would be nicer if I didn't have to.

**Comment by gparant** [comment]
Hello @phoegon 

There is setting of WorkAdventure where you can define if you want to keep open or not camera / microphone when you don't have the WorkAdventure tab focused.

Also, you can check "notification" to receive a browser notification when one person starts a spontaneous discussion 🚀 

Check the following example:

<img width="1325" alt="Screenshot 2023-01-06 at 13 32 45" src="https://user-images.githubusercontent.com/23724509/211013613-bc358617-73b9-4e2e-b468-dc32aa83a2ac.png">
<img width="1503" alt="Screenshot 2023-01-06 at 13 32 29" src="https://user-images.githubusercontent.com/23724509/211013616-8148abf4-d0b6-45fa-b334-9b2fca2ba22f.png">

**Comment by phoegon** [comment]
I have it set (to off), but whenever a colleague steps next to me the camera gets activated even when I'm not using workadventure.
<img width="783" alt="Screen Region 2023-01-06 at 14 57 36" src="https://user-images.githubusercontent.com/6347864/211026647-22bccbf2-2d3a-47b0-b948-38dd8d86d071.png">

**Comment by gparant** [comment]
What is your navigator?
And have you a virtual space that we could test and see the problem?

**Comment by gparant** [comment]
Sorry about the close, was click mistake 😭

**Comment by phoegon** [comment]
Chrome on macOS, and as a pwa.  

our instance is hosted internally. I can try to reproduce on a Public instance with a colleague on Monday.

**Comment by gparant** [comment]
Ok, no problem, you might test on our virtual office https://play.staging.workadventu.re/@/tcm/workadventure/wa-village and come to meet us (david or me greg p).

Just in case, in can't reproduce it on our environment, I've tested with the PWA on our virtual office and works fine: (https://play.staging.workadventu.re/@/tcm/workadventure/wa-village).

https://user-images.githubusercontent.com/23724509/211037332-13c8ca87-b00b-4944-8a0d-c9dd80324275.mov

**Comment by phoegon** [comment]
Hi, 

thanks for the video. I managed to reproduce it on https://play.staging.workadventu.re/@/tcm/workadventure/wa-village

By comparing it to your video the issue seems to be that my Woka doesn't seem to enter away mode, neither in Chrome nor Safari.

https://user-images.githubusercontent.com/6347864/211273524-41b9794a-ff81-47b4-b308-0ba1e1fa2f0d.mp4

In case it's relevant: I'm running macOS Ventura (13.0) on a Apple M1 Max

**Comment by moufmouf** [comment]
Ok, I think I know what's wrong.

As long as the window is visible, WorkAdventure considers you are not "away".
To be "away", you need to minimize the window or you need to open a new tab.

If you do so, this should work.

Ideally, we would need a manual control over the "away" mode, but we do not have that yet.

**Comment by phoegon** [comment]
You're right, opening a new tab works in the browser, but putting a window over it doesn't. 

For the PWA it doesn't work as quickly as in your recording, but it happens eventually and minimizing the PWA window get's me the desired result immediately, which is a perfectly fine workaround for me.

**Comment by phoegon** [comment]
Btw: What do you want me to do with this issue? Ideally I'd like it if the away mode worked even if the tab stays on top, but I'm happy with the workaround.

Do you want me to close it or should I turn it into an issue (which you may never be able to close?) about away mode not working if the tab stays active even though the browser is in the background?
